### DISCUSSION TITLE
New css type accent-color

### DISCUSSION THREAD

**Discussion by DanielBailey-web** [tailwindlabs/tailwindcss#5369] [Ideas]
Although the support for it is still low (only on firefox, chrome, and android), accent-color was added recently. It would be great if it was part of tailwind. I assume it hasn't already been added (I couldn't find it in the docs, or by searching this repo)

Here is the MDN docs
https://developer.mozilla.org/en-US/docs/Web/CSS/accent-color

**Comment by lukewarlow** [comment]
https://github.com/lukewarlow/tailwind-accent-color - Here's a plugin to use this in your Tailwind projects.

I believe it probably won't be added to Core until Safari supports it. Which will probably be next April roughly, based on past release cadence.

**Reply by lukewarlow** [reply]
https://github.com/tailwindlabs/tailwindcss/pull/5387 - I've put together an MR to get them added to Tailwind directly.

**Reply by DanielBailey-web** [reply]
Wow that's awesome! It would be cool if there was a @import for tailwind for bleeding edge features like this so we don't have to wait for safari all the time haha. 

Like this or something:
```css
@import "tailwindcss/novel";
```
So that it wouldn't be in base, but also could get some early use.

**Reply by lukewarlow** [reply]
The PR has now been merged, so it should be coming in Tailwind 3!
